src/deepsparse/utils/onnx.py

```diff
@@ -204,17 +246,9 @@ def override_onnx_batch_size(onnx_filepath: str, batch_size: int) -> str:
         external_input.type.tensor_type.shape.dim[0].dim_value = batch_size
 
     # Save modified model, this will be cleaned up when context is exited
-    shaped_model = tempfile.NamedTemporaryFile(mode="w", delete=False)
-    onnx.save(model, shaped_model.name)
-
-    try:
-        yield shaped_model.name
-    finally:
-        os.unlink(shaped_model.name)
-        shaped_model.close()
+    return save_onnx(model)
 
 
-@contextlib.contextmanager
 def override_onnx_input_shapes(
```

Comment:
hmm do all usages of this function that use it as a context manager need to be changed to normal functions? (same question for override_onnx_batch_size)